testsuite/docker/opentracing-jaeger/src/it/java/io/thorntail/testsuite/opentracing/jaeger/OpenTracingJaegerAppIT.java

```diff
@@ -64,42 +65,43 @@ public void test() throws Exception {
         System.err.println(tree);
         System.err.println("---");
 
-        assertThat(tree).hasRootSpans(2);
+        // TODO what is the first root span?
```

Comment:
The other root span is, which is a bit weir. I think it's comming from jax-rs instrumentation and it should be linked to jax-rs span (maybe bug in `SpanNode` as it is not parsing `FOLLWOS_FROM` reference type).

> [ spanId: 938a897918d5ad5c, operation:deserialize, tags: {entity.type=java.lang.String, media.type=*/*}]